code/dynamic_programming/src/longest_repeating_subsequence/longest_repeating_subsequence.cpp

```diff
@@ -0,0 +1,51 @@
+// Part of Cosmos by OpenGenus Foundation
+
+//dynamic programming || Longest repeating subsequence
+
+#include <iostream>
+#include <cstring>
+
+using namespace std;
+
+const int MAX = 1010;
+
+int dp[MAX][MAX];
```

Comment:
If possible, use dynamic data structure, not pre-defined array.
Because all of files in the project may be part of one's libs, not just for practice.